cmd/experimental/kueue-viz/frontend/src/Dashboard.js

```diff
@@ -0,0 +1,247 @@
+import { Grid, Paper, Table, TableBody, TableCell, TableContainer, TableHead, TableRow, Tooltip, Typography, IconButton, Collapse, Button } from '@mui/material';
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { toast, ToastContainer } from 'react-toastify';
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+import KeyboardArrowUpIcon from '@mui/icons-material/KeyboardArrowUp';
+import useWebSocket from './useWebSocket';
+import './App.css';
+import { AccessTime, Check, CheckBox, CheckCircle } from '@mui/icons-material';
+
+const Dashboard = () => {
+  const [queues, setQueues] = useState([]);
+  const [workloads, setWorkloads] = useState([]);
+  const [expandedRows, setExpandedRows] = useState({});
+  const [expandAll, setExpandAll] = useState(false);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+  const [workloadsByUid, setWorkloadsByUid] = useState({});
+
+  const { data: kueueData, error: kueueError } = useWebSocket('/ws/workloads/dashboard');
+
+  useEffect(() => {
+    if (kueueData) {
+      setQueues(kueueData.queues.items || []);
```

Comment:
```suggestion
      setQueues(kueueData.queues?.items || []);
```
Also here is the same problem.